pkg/xen-tools/initrd/init-initrd

```diff
@@ -80,6 +80,9 @@ echo "Executing /mount_disk.sh"
 # Commence launch sequence
 source /mnt/environment
 
+echo "Run acpid daemon"
+acpid -l /acpid.log
```

Comment:
Do we really need to keep acpid logging? In fact, I'd rather it log to the console frankly.

Reply:
Without this flag it try to save logs into /var/log/acpid.log (and it not exists), so I choose this one.

Reply:
I see. Just for kicks, can you also please try one of those:
    * acpid -l -
    * acpid -l /proc/self/fd/1

Reply:
Variant with `acpid -l /proc/self/fd/1` works for me. Changed.